Common/ThreadPool.h

```diff
@@ -40,7 +40,8 @@ class ThreadPool
 		_priority(0)
 	{
 		size_t nThreads = NCPUS();
-		
+		if(nThreads == 0)
+			nThreads = 1;
```

Comment:
Does this mean the default number of threads becomes 1? I think the number of CPUs was the previous default (and is probably a more sensible default).

Reply:
Not quite: it only becomes one when NCPUS() returns zero (this can maybe happen when all CPUs are scheduled for another task, since NCPUS() takes affinity into account). However, this overload of the constructor is never used in DP3, only the one with explicit nthread count is used. I've submitted some additional comments.

Reply:
Oh maybe you've missed the `if` statement above the nThreads = 1...